topics/ecology/tutorials/lotus2-identifying-fungi/workflows/lotus2-fungi-workflow.ga

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
Please run through the workflow best practices (https://training.galaxyproject.org/training-material/faqs/galaxy/workflows_best_practices.html) and be sure you've added yourself (or the appropriate person(s)) as creators to the workflow to help us track provenance :)